pytest_fixtures/broker.py

```diff
@@ -0,0 +1,41 @@
+import pytest
+from broker.broker import VMBroker
+
+from robottelo.constants import BROKER_RHEL77
+from robottelo.hosts import ContentHost
+
+
+@pytest.fixture
+def rhel7_host():
+    """A function-level fixture that provides a host object based on the rhel7 nick"""
+    with VMBroker(nick='rhel7') as host:
+        yield host
+
+
+@pytest.fixture
+def rhel7_contenthost():
+    """A fixture that provides a content host object based on the rhel7 nick"""
+    with VMBroker(nick='rhel7', host_classes={'host': ContentHost}) as host:
+        yield host
+
+
+@pytest.fixture(scope="module")
+def module_rhel77_host():
+    """A module-level fixture that provides a host object"""
+    with VMBroker(**BROKER_RHEL77) as host:
+        yield host
+
+
+@pytest.fixture(scope="module")
+def module_rhel77_contenthost():
+    """A module-level fixture that provides a ContentHost object"""
+    with VMBroker(host_classes={'host': ContentHost}, **BROKER_RHEL77) as host:
+        yield host
+
+
+@pytest.fixture(scope="class")
+def unittest_rhel77_contenthost(request):
```

Comment:
Should we even add unittest class support if we will be converting to pytest soon? If so, shouldn't we also add a fixture to support unittest_rhel77_host()?

Reply:
This is something we could debate. It may still take a while for everything to be converted away from unittest style.

Reply:
Let's say we keep unittest fixtures in, Do we have a class-level fixture that provides a Host object?

Reply:
this is that fixture